projects/Open-Rival-test/src/tests/TestWalkerComponent.cpp

```diff
@@ -6,6 +6,7 @@
 #include "game/Pathfinding.h"
 #include "game/World.h"
 #include "utils/TimeUtils.h"
+#include <array>
```

Comment:
Can you format the file?

The project is configured with some auto-formatting settings which includes the ordering of includes.

I use an extension called [Format Document on Save](https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave) so it's entirely automatic.